File: android/src/main/java/com/valentingrigorean/arcgis_maps_flutter/map/PolylineControllerSink.java
package com.valentingrigorean.arcgis_maps_flutter.map;

import com.esri.arcgisruntime.symbology.SimpleLineSymbol;

public interface PolylineControllerSink extends GraphicControllerSink {
    void setColor(int color);

    void setWidth(float width);

    void setStyle(SimpleLineSymbol.Style style);

    void setAntialias(boolean antialias);
}

File: android/src/main/java/com/valentingrigorean/arcgis_maps_flutter/map/PolylineControllerSink.kt
package com.valentingrigorean.arcgis_maps_flutter.map

import com.esri.arcgisruntime.symbology.SimpleLineSymbol

interface PolylineControllerSink : GraphicControllerSink {
    fun setColor(color: Int)
    fun setWidth(width: Float)
    fun setStyle(style: SimpleLineSymbol.Style?)
    fun setAntialias(antialias: Boolean)
}